emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -177,7 +177,7 @@ install_upgrade_telepresence() {
         fi
     fi    
     if [ $install_telepresence = true ]; then
-        sudo curl -fL https://app.getambassador.io/download/tel2/${OS}/${ARCH}/2.11.0/telepresence -o /usr/local/bin/telepresence
+        sudo curl -fL https://app.getambassador.io/download/tel2/${OS}/${ARCH}/2.12.1/telepresence -o /usr/local/bin/telepresence
```

Comment:
I assume you've tested it and it works ? :P

Reply:
@knlambert 
<img width="876" alt="image" src="https://user-images.githubusercontent.com/40805575/227234026-175a6ff1-d2a3-43ae-bc1e-488897b4b226.png">